packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -814,7 +814,6 @@ export type CameraButtonSelector = (state: CallClientState, props: CallingBaseSe
 
 // @public
 export interface CameraButtonStrings {
-    cameraButtonSplitRoleDescription?: string;
```

Comment:
JAMES BOT: THIS IMPACTS THE STABLE API FILE, HAVE YOU CONFIRMED THIS IS NOT PART OF ANY STABLE RELEASE?

Reply:
Yes I believe it did go out when we released CallWithChatComposite. How would we go about removing this string?

Reply:
First check on stable if this issue repros (git checkout the 1.2.0 tag and run locally), if not do we have to remove it?